App/Store/StoreManager.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace App\Store;
+
+use App\{Enums\Status, Interface\FileManagerInterface, Model\Collection, Model\Fast};
+use DateTime;
+use Exception;
+
+
+class StoreManager implements FileManagerInterface
+{
+
+    protected string $file = "./store.json";
```

Comment:
It would be good if you are getting the file name from a .env file so you can change it whenever you want